### PR TITLE
CLI user-info JSON option

### LINKED ISSUE #2967: There is any way to remove user older than 365 days ?
Hello 

I host a FreshRSS instance and the size of the database is constantly increasing (logic).
But, I suspect some users are registering to test FreshRSS and import some (or many) and there are no remove there accounts. (I have more user registered than access on front web).
So FreshRSS get rss for unused user, consume ressource and use space in DB.

I research a way to remove users who don't connect on front web or who don't get there rss feed after x days.
It is possible ?

Thank :)

### PR DESCRIPTION
**Pull request by Alkarex**
Add a JSON user-info output
https://github.com/FreshRSS/FreshRSS/issues/2967#issuecomment-627441744

### CODE REVIEW

**Comment by Alkarex** [comment]
Example:

```json
[
  {
    "is_default": true,
    "user": "alex",
    "last_user_activity": "2020-05-12T17:45:29Z",
    "database_size": 422281216,
    "categories": "20",
    "feeds": 196,
    "reads": 128747,
    "unreads": 833,
    "favourites": 274,
    "tags": "18",
    "lang": "fr",
    "mail_login": "alexandre@example.net"
  },
  {
    "is_default": false,
    "user": "alice",
    "last_user_activity": "2020-01-06T20:40:41Z",
    "database_size": 221184,
    "categories": "1",
    "feeds": 1,
    "reads": 2,
    "unreads": 9,
    "favourites": 0,
    "tags": "0",
    "lang": "fr",
    "mail_login": ""
  }
]
```

File: cli/user-info.php [hunk 1]
```diff
@@ -7,11 +7,12 @@
 $params = array(
 	'user:',
 	'header',
+	'json',
 );
 $options = getopt('h', $params);
 
 if (!validateOptions($argv, $params)) {
-	fail('Usage: ' . basename(__FILE__) . ' (-h --header --user username --user username …)');
+	fail('Usage: ' . basename(__FILE__) . ' (-h --header --json--user username --user username …)');
```
Comment (Frenzie) [inline cli/user-info.php]: typo

**Review by Frenzie — APPROVED** [review]
lgtm besides the typo

**Comment by Alkarex** [comment]
I have added a few examples

**Pull request merged**